iptables/iptables.go

```diff
@@ -220,6 +220,20 @@ func (ipt *IPTables) DeleteIfExists(table, chain string, rulespec ...string) err
 	return err
 }
 
+// GetRuleById return rule specifing its Id
+func (ipt *IPTables) GetRuleById(table, chain string, ruleId int) (string, error) {
```

Comment:
`iptables -S` takes an optional `rulenum`, e.g.

```
$ iptables -S FORWARD 1
-A FORWARD -j DOCKER-USER
```

Given that iptables is 1-indexed, so the library should match this.